### PR TITLE
[zh]tranlate Kubernetes Release Cycle in release section

### PR DESCRIPTION
**Pull request by howieyuen**
/depend https://github.com/kubernetes/website/pull/34049

related issue: https://github.com/kubernetes/website/issues/30096

### CODE REVIEW

File: content/zh/releases/release.md [hunk 1]
```diff
@@ -0,0 +1,659 @@
...
+<!-- 
+# Targeting enhancements, Issues and PRs to Release Milestones
+
+This document is focused on Kubernetes developers and contributors who need to
+create an enhancement, issue, or pull request which targets a specific release
+milestone.
+-->
+# 针对发布里程碑的特性增强、Issue 和 PR  {#targeting-enhancements-issues-and-prs-to-release-milestones}
+
+本文档重点关注需要创建针对特定发布里程碑的特性增强、问题或拉取请求的 Kubernetes 开发人员和贡献者。
```
Comment (TinySong) [inline content/zh/releases/release.md:31]:
```suggestion
本文档重点是面向于那些需要创建针对特定发布里程碑的特性增强、问题或拉取请求的 Kubernetes 开发人员和贡献者。
```

File: content/zh/releases/release.md [hunk 2]
```diff
@@ -0,0 +1,659 @@
...
+
+- **问题所有者**：将问题移至发布里程碑的创建者、委托人和用户
+
+- **发布小组**：每个 Kubernetes 版本都有一个团队来执行[这里][release-team]所描述的项目管理任务。
+
+  可以在[此处](https://git.k8s.io/sig-release/releases/)找到与任何给定版本相关联的团队的联系信息。
+
+- **Y 天**：指工作日
+
+- **增强**：参见“[我的工作是特性增强吗？](https://git.k8s.io/enhancements/README.md#is-my-thing-an-enhancement)”
```
Comment (Sea-n) [inline content/zh/releases/release.md:186]:
```suggestion
- **增强**：参见“[我的改進是特性增强吗？](https://git.k8s.io/enhancements/README.md#is-my-thing-an-enhancement)”
```

File: content/zh/releases/release.md [hunk 3]
```diff
@@ -0,0 +1,659 @@
...
+
+<!-- 
+With ongoing feature definition through the year, some set of items will bubble
+up as targeting a given release. **[Enhancements Freeze][enhancements-freeze]**
+starts ~4 weeks into release cycle. By this point all intended feature work for
+the given release has been defined in suitable planning artifacts in
+conjunction with the Release Team's [Enhancements Lead](https://git.k8s.io/sig-release/release-team/role-handbooks/enhancements/README.md).
+-->
+随着全年功能定义的不断进行，某些项目将冒泡作为给定版本的目标。
+在发布周期约 4 周后开始**[冻结特性增强][enhancements-freeze]**。
```
Comment (Sea-n) [inline content/zh/releases/release.md:287]: We shouldn't add space between Mandarin words, but the `**bold**` is not parsed.
```suggestion
在发布周期约 4 周后开始 **[冻结特性增强][enhancements-freeze]**。
```

File: content/zh/releases/release.md [hunk 4]
```diff
@@ -0,0 +1,659 @@
...
+例如要添加指示 SIG Storage 负责的标签，请使用 `/sig storage` 进行评论。
+
+<!-- 
+### Priority Label
+
+Priority labels are used to determine an escalation path before moving issues
+out of the release milestone. They are also used to determine whether or not a
+release should be blocked on the resolution of the issue.
+-->
+### 优先标签  {#priority-label}
```
Comment (Sea-n) [inline content/zh/releases/release.md:579]:
```suggestion
### 优先度标签  {#priority-label}
```

**Review by Sea-n — COMMENTED** [review]
Overall LGTM, here is the [Netlify preview](https://deploy-preview-34050--kubernetes-io-main-staging.netlify.app/zh/releases/release).

The image is not shown, I will open another to fix it for English version.

File: content/zh/releases/release.md [hunk 5]
```diff
@@ -0,0 +1,659 @@
+---
+title: Kubernetes 发布周期
+type: docs
+auto*generated: true
```
[the remark this answers is not in the document]
Comment (howieyuen) [reply content/zh/releases/release.md:4]: good catch

File: content/zh/releases/release.md [hunk 6]
```diff
@@ -0,0 +1,659 @@
...
+up as targeting a given release. **[Enhancements Freeze][enhancements-freeze]**
+starts ~4 weeks into release cycle. By this point all intended feature work for
+the given release has been defined in suitable planning artifacts in
+conjunction with the Release Team's [Enhancements Lead](https://git.k8s.io/sig-release/release-team/role-handbooks/enhancements/README.md).
+-->
+随着全年功能定义的不断进行，某些项目将冒泡作为给定版本的目标。
+在发布周期约 4 周后开始**[冻结特性增强][enhancements-freeze]**。
+至此，针对给定版本的所有预期功能工作都已与发布团队的
+[特性增强负责人](https://git.k8s.io/sig-release/release-team/role-handbooks/enhancements/README.md
+一起在合适的规划工件中定义。
```
[the remark this answers is not in the document]
Comment (howieyuen) [reply content/zh/releases/release.md:290]: I tried, merged line is too long and my screen (16 inc mac) can not keep it one line...

File: content/zh/releases/release.md [hunk 7]
```diff
@@ -0,0 +1,659 @@
...
+- `kind/api-change`：添加、删除或更改 API
+- `kind/bug`：修复了一个新发现的错误
+- `kind/cleanup`：添加测试、重构、修复旧错误
+- `kind/design`：与设计相关
+- `kind/documentation`：添加文档
+- `kind/failing-test`：CI 测试用例一直失败
+- `kind/feature`：新功能
+- `kind/flake`：CI 测试用例显示间歇性故障
+
+[cherry-picks]: /contributors/devel/sig-release/cherry-picks.md
```
[the remark this answers is not in the document]
Comment (howieyuen) [reply content/zh/releases/release.md:652]: yes, so I use the latest url, your suggestion make a back-off

File: content/zh/releases/release.md (hunk 3 above)
Comment (howieyuen) [reply content/zh/releases/release.md:287]: nice catch, thanks

File: content/zh/releases/release.md (hunk 4 above)
Comment (howieyuen) [reply content/zh/releases/release.md:579]: I prefer 优先级标签

File: content/zh/releases/release.md [hunk 8]
```diff
@@ -0,0 +1,659 @@
...
+  - [Milestone Maintainers](#milestone-maintainers)
+  - [Feature additions](#feature-additions)
+  - [Issue additions](#issue-additions)
+  - [PR Additions](#pr-additions)
+- [Other Required Labels](#other-required-labels)
+  - [SIG Owner Label](#sig-owner-label)
+  - [Priority Label](#priority-label)
+  - [Issue/PR Kind Label](#issuepr-kind-label)
+-->
+- [TL;DR](#tldr)
```
[the remark this answers is not in the document]
Comment (howieyuen) [reply content/zh/releases/release.md:50]: TL;DR means too long; didn’t read, “太长，未读”

File: content/zh/releases/release.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (Sea-n) [reply content/zh/releases/release.md:653]: You are right, sorry for missed #34049. 😅

File: content/zh/releases/release.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (howieyuen) [reply content/zh/releases/release.md:50]: @tengqm what's your opinion?

File: content/zh/releases/release.md [hunk 9]
```diff
@@ -0,0 +1,660 @@
...
```
Comment (Sea-n) [inline content/zh/releases/release.md:50]: I still think `長話短說` could be better in this context. But it can be discuss later.

**Review by Sea-n — COMMENTED** [review]
Thanks a lot! This PR looks good to me! 🎉
/lgtm

**Pull request merged**